src/Divante/ObjectMapperBundle/Mapper/Strategy/ClassificationStore/MapQuantityValueKey.php

```diff
@@ -17,7 +17,7 @@ class MapQuantityValueKey extends AbstractMapKeyStrategy
 {
 
     const ALLOWED_TYPES_ARRAY = ['inputQuantityValue', 'quantityValue'];
-    const TYPE = 'text';
+    const TYPE = 'quantityValue';
```

Comment:
Type is used as a hint in Magento system. It does not support quantity value type so we map it into text type